### PR TITLE
Domains: normalize ccTLD Redux data

### PR DESCRIPTION
**Pull request by hewsut**
Right now we jumble all collected ccTLD data together in one big meta field. That bad — we get data collisions among other issues.

This change normalizes the structure by TLD to eliminate those concerns.

See the broken aspect of #22097 for background.

Backend companion is D10103-code.

### CODE REVIEW

File: client/components/domains/registrant-extra-info/uk-form.jsx [hunk 1]
```diff
@@ -89,13 +90,17 @@ export class RegistrantExtraInfoUkForm extends React.PureComponent {
 		}
 
 		this.props.updateContactDetailsCache( {
-			extra: pick( defaultValues, neededRequiredDetails ),
+			extra: {
+				uk: pick( defaultValues, neededRequiredDetails ),
```
[the remark this answers is not in the document]
Comment (hewsut) [reply client/components/domains/registrant-extra-info/uk-form.jsx]: I like it. I'd suggest adding that as a new function and swapping it into place everywhere in a follow up PR. Just to get some momentum going again here.

**Comment by deBhal** [comment]
Rebased

**Comment by deBhal** [comment]
The rebase brought the new schema code into play, so I've updated front and back to allow for that, and successfully bought a cart full of tlds 🎉 

LGTM, but I've spent most of the day on it, so I've pinged Team Cobalt to have a look at it overnight, but I'm happy to go ahead and merge tomorrow if they're too busy.

**Comment by aidvu** [comment]
Good job with this!

**Pull request merged**